resources/views/checkout/overview.blade.php

```diff
@@ -5,19 +5,19 @@
 @section('robots', 'NOINDEX,NOFOLLOW')
 
 @section('content')
-    <checkout v-cloak v-slot="{ checkout, cart, hasItems, save, goToStep }">
+    <checkout v-cloak v-slot="{ checkout, cart, hasItems, save, goToStep }" :set="checkout.step = Math.max(checkout.step, 2)">
```

Comment:
Why do we need this? You want to start at step 2 instead of 1? In that case the `checkout_steps` from the Rapidez config should be changed right? The cart is currently just included there to get the cart in the progressbar. Put this change in the installation instructions and we don't need this I think?

Reply:
In theory, this would be the way to go. It doesn't really end up working however, as there are [quite a few places in the Rapidez core](https://github.com/rapidez/core/blob/master/resources/js/components/Checkout/Login.vue#L76) where step numbers are hardcoded which means deviating from the numbering in a package like this is not really possible. We're essentially forced for this sort of theme to set the starting step number to 2 because of how the core checkout works.

Reply:
Alright, good moment to fix that? `step++` instead of hardcoded?

Reply:
Not sure how that would work in the current situation (the specific line I pointed out should probably not even exist at all). I feel like this would be a part of the planned big checkout refactor.

Reply:
Thoughts @indykoning?

Reply:
While i do agree that the true fix should be the big checkout refactor, it will be a big and breaking change so it's good to try and make the current checkout suit us best in the meantime.

I'm afraid that if we do not do anything now we will forget about it soon enough, spreading that code smell across multiple repositories making it more difficult to fix in the future

Reply:
In coordination with Indy I've made an issue to patch this in the core checkout, so we can solve this at the source without needing the whole checkout refactor yet.